pyctcdecode/language_model.py

```diff
@@ -271,6 +287,82 @@ def score(
         lm_score = self.alpha * lm_score * LOG_BASE_CHANGE_FACTOR + self.beta
         return lm_score, end_state
 
+    @property
```

Comment:
this needs a bit more documentation and a small refactor to make there be only one source of file names and serialization schemes, but looking to see if this is approximately the right direction